jack.py

```diff
@@ -842,6 +842,8 @@ def set_shutdown_callback(self, callback):
                shutdown callback and has to be called outside of the
                callback context.
 
+            .. warning:: Same as with most jack callbacks, no functions
+               that iteract with jack daemon should be used here.
```

Comment:
Please add an empty line before the closing `"""`, as this seems to be the NumPy style.